website/content/docs/k8s/k8s-cli.mdx

```diff
@@ -89,38 +92,378 @@ The following example commands install Consul on Kubernetes using custom values,
+### `proxy`
 
-The `uninstall` command removes Consul from Kubernetes.
+The `proxy` command exposes two subcommands for interacting proxies managed by
+Consul in your Kubernetes Cluster.
+
+- [`proxy list`](#proxy-list): List all Pods running proxies managed by Consul.
+- [`proxy read`](#proxy-read): Inspect the Envoy configuration for a given Pod.
+
+### `proxy list`
 
 ```shell-session
-$ consul-k8s uninstall <OPTIONS>
+$ consul-k8s proxy list <OPTIONS>
+```
+
+| Flag                                 | Description                                           | Default                                                                                                                |
+| ------------------------------------ | ----------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------- |
+| <nobr>`-all-namespaces`, `-A`</nobr> | `Boolean` List pods in all Kubernetes namespaces.     | `false`                                                                                                                |
+| <nobr>`-namespace`, `-n`</nobr>      | `String` The Kubernetes namespace to list proxies in. | Current [kubeconfig](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/) namespace. |
+
+See [Global Options](#global-options) for additional options that you can use
+when installing Consul on Kubernetes.
+
+This command will list proxies alongside their `Type`. Types of proxies include
+
+- `Sidecar`: these will be the majority of pods in the cluster. They run the
+  proxy in a sidecar pattern to network the pod as a service in the mesh.
+- `API Gateway`: these pods run a proxy to manage connections with networks
+  outside of the Consul cluster. [Read more about API gateways](/docs/api-gateway).
+- `Ingress Gateway`: these pods run a proxy to manage ingress into the
+  Kubernetes cluster. [Read more about ingress gateways](/docs/k8s/connect/ingress-gateways).
+- `Terminating Gateway`: these pods run a proxy to control connections to
+  external services. [Read more about terminating gateways](/docs/k8s/connect/terminating-gateways).
+- `Mesh Gateway`: these pods run a proxy to manage connections between
+  Consul clusters connected using mesh federation. [Read more about Consul Mesh Federation](/docs/k8s/installation/multi-cluster/kubernetes).
+
+#### Example Commands
+
+Display all pods in the current Kubernetes namespace which run proxies managed
+by Consul.
+
+```shell-session
+$ consul-k8s proxy list
+```
+
+```
+Namespace: default
+
+Name                     	Type
+backend-658b679b45-d5xlb 	Sidecar
+client-767ccfc8f9-6f6gx  	Sidecar
+client-767ccfc8f9-f8nsn  	Sidecar
+client-767ccfc8f9-ggrtx  	Sidecar
+frontend-676564547c-v2mfq	Sidecar
+```
+
+Display all pods in the `consul` Kubernetes namespace which run proxies managed
+by Consul.
+
+```shell-session
+$ consul-k8s proxy list -n consul
+```
+
+```
```

Comment:
you can include outputs in the codeblocks, the copy will only copy the first line

```suggestion
```